site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -33,6 +33,13 @@
                 </div>
             </div>
         </div> 
+        <div class="row">
+            <div class="card col-md-12" style="margin:0px">
+                <div class="content"style="margin:0px">
+                    Enter any file wildcards that should not be viewable by students. Seperate wildcards with commas <br>
+                    <input type="text" id="files_to_omit" name = "files_to_omit" class="auto_save" placeholder="" value = {{hidden_files}}>
```

Comment:
use dashes for id's, names', and classes instead of underscores, e.g: `files-to-omit`

Reply:
Changing to files-to-omit means the name of the setters and getters would have to be Files-To-Omit setFiles-to-omit()/ getFiles-to-omit(). I think it would be better to just use files_to_omit.

Reply:
CSS ids should use dashes see the style guide : https://submitty.org/developer/coding_style_guide/css,
its fine to leave for now if there's JS logic depending on the ID using underscores but in the future new code should stick to using dashes instead